corehq/apps/users/static/users/js/edit_commcare_user.js

```diff
@@ -56,7 +56,7 @@ hqDefine('users/js/edit_commcare_user', function() {
         return false;
     });
     if (!initial_page_data('is_currently_logged_in_user')) {
-        function DeleteUserButtonModel() {
+        var DeleteUserButtonModel = function() {
```

Comment:
Is this a lint thing or personal preference? (personally I prefer `function name() {` syntax).

Reply:
It's a lint thing - [no inner declarations](https://eslint.org/docs/rules/no-inner-declarations). I don't feel strongly either way, but we should turn the rule off if we support both styles.